app/Subscriber/AuthSubscriber.php

```diff
@@ -95,10 +95,11 @@ public function onLoginFailure(AuthFailureEvent $event)
     {
         $this->logger->debug('Subscriber executed: '.__METHOD__);
         $username = $event->getUsername();
-
+        $ipAddress = $this->request->getIpAddress();
+        
         if (! empty($username)) {
             // log login failure in web server log to allow fail2ban usage
-            error_log('Kanboard: user '.$username.' authentication failure');
+            error_log('Kanboard: user '.$username.' authentication failure with IPAddress: ' $ipAddress);
```

Comment:
This is not valid in PHP. You must add a dot `.` to concat strings.

And the wording could be changed, for example `IP address:` instead of `IPAddress:`.